api/team.go

```diff
@@ -723,6 +730,28 @@ func getTeamByName(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getMyTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
+	if result := <-Srv.Store.Team().GetTeamsForUser(c.Session.UserId); result.Err != nil {
```

Comment:
Totally forgot about that, I'll add the cache read here if it exists I'll return that